test/allowlist_for_publicAPI.json

```diff
@@ -375,6 +375,7 @@
     "GradBucket",
     "HashStore",
     "Logger",
+    "namedtuple",
```

Comment:
You should not add new entries to this file in general.
You will need to properly only expose the public functions and make non-public ones properly private as per the recommendation from the failing test.

Reply:
This is removed for the next iteration